.reek.yml

```diff
@@ -0,0 +1,9 @@
+detectors:
+  IrresponsibleModule:
```

Comment:
Don't touch files, which is not in your folder, return all such files in original state